src/test/java/com/maxmind/geoip2/model/CountryResponseTest.java

```diff
@@ -50,6 +50,7 @@
                     "      \"names\" : {" +
                     "         \"en\" : \"United Kingdom\"" +
                     "      }," +
+                    "      \"is_in_european_union\": true," +
```

Comment:
Given that you added fixture files, maybe it would be nice to move this one to those too (but not necessary).